src/gui/gtk.c

```diff
@@ -1439,7 +1439,7 @@ int dt_gui_gtk_init(dt_gui_gtk_t *gui)
   dt_accel_register_global(NC_("accel", "toggle header"), GDK_KEY_h, GDK_CONTROL_MASK);
   dt_accel_connect_global("toggle header", g_cclosure_new(G_CALLBACK(_toggle_header_accel_callback), NULL, NULL));
 
-  dt_accel_register_global(NC_("accel", "toggle filmstrip and timeline"), GDK_KEY_f, GDK_CONTROL_MASK);
+  dt_accel_register_global(NC_("accel", "toggle filmstrip and timeline"), GDK_KEY_b, GDK_CONTROL_MASK);
```

Comment:
That combination is already assigned to color assessment mode : 

```
  // toggle ISO 12646 color assessment condition
  dt_accel_register_view(self, NC_("accel", "color assessment"), GDK_KEY_b, GDK_CONTROL_MASK);
```

Reply:
In darkroom only. Does that breaks darkroom color assessment ?

Reply:
Hmm, filmstrip could be displayed in darkroom, so if we set this shortcut, this will be in conflict with color assessment.

And changing shortcuts used since years is not something we should do quickly. I think this PR has been merged too quickly.

Reply:
ctrl+f for filmstrip (as f for filmstrip) is logic, but I understand is not compatible with F for fullscreen, so ctrl+f but maybe need to be discussed more to make the good choices in all places.

Reply:
I have reverted this part of the PR for now. No big deal.